docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -134,6 +134,7 @@ It supports the following options:
   - `config.matchFrom` (*'any' | 'start'* [optional]): Defaults to `'any'`.
   - `config.stringify` (*Func* [optional]): Defaults to `JSON.stringify`.
   - `config.trim` (*Boolean* [optional]): Defaults to `false`. Remove trailing spaces.
+  - `config.limit` (*Number* [optional]): Limit the number of suggested options to be shown. For example, if `config.limit` is `10`, only the first `10` matching options are shown. If not set, no limit is applied (This is the default behavior).
```

Comment:
Should we mention a potential use case?
```suggestion
  - `config.limit` (*Number* [optional]): Default to null. Limit the number of suggested options to be shown. For example, if `config.limit` is `100`, only the first `100` matching options are shown. It can be useful if a lot of options match and virtualization wasn't set up.
```